databases/concession_sales_db.txt

```diff
@@ -1,9 +1,4 @@
-Kids Combo,90,
-Popcorn,102,
-Drink,85,
-Hot Dog,63,
-Nachos,110,
-Chicken Strips,85,
-Chocolate Bites,110,
-Twix,19,
-Pizza,5,
+Nachos,67,
+Popcorn,15,
```

Comment:
When I purchase concessions as a user then go to the admin page it doesn't update the sales for me.

Reply:
Functionality added for sales, and similar issues noted in readme.